### PR TITLE
Update documentation/application links in the header.

### PR DESCRIPTION
**Pull request by gijskant**

### CODE REVIEW

File: src/app/modules/gb-side-panel-module/gb-side-panel.component.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <div class="side-panel-container">
   <div class="gb-logo-panel">
-    <img src="../../../assets/icons/glowingbear.png" (click)="goToDocUrl()">
-    <span class="gb-logo-text" (click)="goToDocUrl()">GLOWING BEAR</span>
+    <a [routerLink]="''">
```
Comment (forus) [inline src/app/modules/gb-side-panel-module/gb-side-panel.component.html]: Shouldn't it be `[routerLink]="['/']"` instead?

Comment (forus) [reply src/app/modules/gb-side-panel-module/gb-side-panel.component.html]: I assume empty string means current path?

Comment (gijskant) [reply src/app/modules/gb-side-panel-module/gb-side-panel.component.html]: It refers to the route defined here: https://github.com/thehyve/glowing-bear/blob/dev/src/app/app.routing.ts#L15.

Comment (forus) [reply src/app/modules/gb-side-panel-module/gb-side-panel.component.html]: ah, ok

**Pull request merged**